client/src/components/AlumniMentorship.js

```diff
@@ -600,6 +624,35 @@ class RequestCards extends Component {
         this.setState({display: display})
     }
 
+    async getActionItemsInput(selection) {
+        this.setState({
+            existingActionItems: selection.old,
+            newActionItems: selection.new,
+        });
+    }
+
+    async submitActionItems() {
+        let requests = await makeCall({
+            requestId: this.state.requestDetails._id,
+            existingActionItems: this.state.existingActionItems,
+            newActionItems: this.state.newActionItems
+        }, `/request/actionItems/${this.state}/`, 'patch')
```

Comment:
and pass it as a url parameter here like `/${this.state.requestDetails._id}` (this.state here is an object and cannot be parsed correctly)